vendor/github.com/container-storage-interface/spec/lib/go/csi/v0/csi.pb.go

```diff
@@ -0,0 +1,2671 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Should be old `container-storage-interface/spec/lib/go/csi/csi.pb.go` removed?

Reply:
removed